src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck started working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stopped working");

```

Comment:
Similarly, the message 'Truck stopped working' should be more informative. Consider specifying what the Truck has stopped doing, such as 'Truck has completed transporting cargo and is now idle.'